apps/api/src/config/database/configuration.ts

```diff
@@ -15,6 +15,9 @@ export class DatabaseConfiguration implements TypeOrmOptionsFactory {
       username: this.configService.getOrThrow<string>('DB_USERNAME'),
       password: this.configService.getOrThrow<string>('DB_PASSWORD'),
       database: this.configService.getOrThrow<string>('DB_NAME'),
+      logging: this.configService.getOrThrow<boolean>('LOGGING'),
+      // logger: this.configService.getOrThrow<string>('LOGGER'),
```

Comment:
Remove all commented code. This file should not have any changes by you.

Reply:
Done